microservices/collaboration-service/Dockerfile.collab

```diff
@@ -0,0 +1,13 @@
+FROM node:18
+
+WORKDIR /collab-server
+
+COPY . .
```

Comment:
I think we're missing a `.dockerignore` file the collaboration-service folder -- we don't want to copy our entire `node_modules` folder into the Docker image